forge-game/src/main/java/forge/game/trigger/TriggerDamageDoneOnce.java

```diff
@@ -80,16 +81,16 @@ public int getDamageAmount(Map<Card, Integer> damageMap) {
         return result;
     }
 
-    public Set<Card> getDamageSources(Map<Card, Integer> damageMap) {
-        if (!hasParam("ValidSource")) {
-            return Sets.newHashSet(damageMap.keySet());
-        }
-        Set<Card> result = Sets.newHashSet();
-        for (Card c : damageMap.keySet()) {
-            if (matchesValid(c, getParam("ValidSource").split(","))) {
-                result.add(c);
-            }
+    public CardCollection getDamageSources(Map<Card, Integer> damageMap) {
+    if (!hasParam("ValidSource")) {
+        return new CardCollection(damageMap.keySet());
+    }
+    CardCollection result = new CardCollection();
+    for (Card c : damageMap.keySet()) {
+        if (matchesValid(c, getParam("ValidSource").split(","))) {
+            result.add(c);
```

Comment:
the `}` in lines 92, 94, 96 all need to be tabbed in once more

Reply:
Tried my best to piece the brackets together in pairs to fix this, but my removing the two unnecessary imports beforehand meant I couldn't follow those line numbers! Is the updated file tabbed correctly?